core/src/main/java/org/springframework/security/authentication/ObservationAuthenticationManager.java

```diff
@@ -56,4 +57,11 @@ public Authentication authenticate(Authentication authentication) throws Authent
 		});
 	}
 
+	public ObservationConvention<AuthenticationObservationContext> getConvention() {
```

Comment:
For now, please leave out the getters. Doing so has a tendency to simplify backward compatibility over time, which is nice.

Reply:
Please also remove the other getters.